### PR TITLE
Fix email height + question mark

### PR DESCRIPTION
**Pull request by T-Damer**
- Changed `<Arrow />` component to work more properly
- Added arrow open-close animation
- Hide question mark when closed
- Height is the same as other proofs

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -169,7 +169,11 @@ export default function ({
     >
       {loading && <Loading small={small} />}
       {typeof children === 'string' ? <span>{children}</span> : children}
-      {withArrow && <Arrow disabled={disabled || loading} />}
+      {withArrow && (
+        <div className={width('w-4')}>
```
Comment (T-Damer) [inline src/components/Button.tsx:173]: The arrow has different sizes, so I removed `width` and `height` from `svg` and wrapped it, where we use it

File: src/components/proofs/EmailProof.tsx [hunk 2]
```diff
@@ -11,28 +11,42 @@ import ToolTip from 'components/ToolTip'
...
   backgroundImage('bg-gradient-to-r'),
   gradientColorStops('from-secondary', 'to-accent'),
   display('flex'),
   alignItems('items-center'),
-  space('space-x-2'),
-  fontWeight('font-bold')
+  space('space-x-2')
+)
+
+const getStartedText = classnames(
```
Comment (T-Damer) [inline src/components/proofs/EmailProof.tsx:45]: Text was a little different

File: src/components/proofs/EmailProof.tsx [hunk 3]
```diff
@@ -99,38 +119,51 @@ export default function () {
...
-                    onClick={resetEmail}
-                  >
-                    re-enter email.
-                  </button>
-                </>
-              ) : (
-                `Add your work email and we’ll send you a token for that email. Then, use the token here to create zk proof.`
-              )}
-            </BadgeText>
+            <div className={margin('mt-4')}>
```
Comment (T-Damer) [inline src/components/proofs/EmailProof.tsx:149]: After I changed the height of the container the spacing broke, so I added margin-top

File: src/components/Button.tsx (hunk 1 above)
Comment (T-Damer) [inline src/components/Button.tsx:173]: Again wrapped it to use proper size

File: src/components/landing/DownArrows.tsx [hunk 4]
```diff
@@ -1,24 +1,26 @@
...
+  width('w-6')
 )
 
 export default function () {
   return (
     <div className={arrowsBox}>
-      <Arrow turnDown disabled />
-      <Arrow turnDown disabled />
-      <Arrow turnDown disabled />
+      <Arrow pulseDisabled />
```
Comment (T-Damer) [inline src/components/landing/DownArrows.tsx:21]: Now it's `turnDown` by default

File: src/components/proofs/EmailProof.tsx [hunk 5]
```diff
@@ -52,9 +66,15 @@ const emailTitleLeft = classnames(
...
+const revealAnimation = (open: boolean) =>
+  classnames(
+    animation(open ? 'animate-reveal' : 'animate-unreveal'),
+    visibility(open ? 'visible' : 'invisible')
+  )
+
 export default function () {
   const [loading, setLoading] = useState(false)
-  const [open, setOpen] = useState(true)
+  const [open, setOpen] = useState(false)
```
Comment (T-Damer) [inline src/components/proofs/EmailProof.tsx:77]: It's closed by default in designs

File: src/components/proofs/EmailProof.tsx [hunk 6]
```diff
@@ -99,38 +119,51 @@ export default function () {
   return (
     <Line breakWords>
       <div className={proofLineContainer}>
-        <ToolTip position="bottom" text={popoverText} fitContainer>
+        <ToolTip
+          position="bottom"
+          text={popoverText}
+          fitContainer
+          disabled={!open}
```
Comment (T-Damer) [inline src/components/proofs/EmailProof.tsx:126]: We don't want to show the pop-up when it's closed

File: src/components/proofs/EmailProof.tsx [hunk 7]
```diff
@@ -99,38 +119,51 @@ export default function () {
...
+        >
           <div className={emailTitleContainer}>
             <div className={emailTitleLeft}>
-              <BodyText small>Work email</BodyText>
-              <QuestionMark small />
+              <ProofText>Work email</ProofText>
+              <div className={revealAnimation(open)}>
```
Comment (T-Damer) [inline src/components/proofs/EmailProof.tsx:131]: Hides the question mark

File: src/components/QuestionMark.tsx [hunk 8]
```diff
@@ -24,9 +24,15 @@ const questionStyles = (small?: boolean) =>
     alignItems('items-center'),
     fontSize(small ? 'text-xs' : 'text-base'),
     fontFamily('font-primary'),
-    cursor('cursor-pointer')
+    cursor(disabled ? undefined : 'cursor-pointer')
```
Comment (T-Damer) [inline src/components/QuestionMark.tsx:27]: The block is invisible, but can be hovered

File: src/components/ToolTip.tsx [hunk 9]
```diff
@@ -78,12 +78,16 @@ export default function ({
...
   children,
+  disabled,
 }: ChildrenProp & {
   text: string
   position: 'top' | 'bottom'
   arrow?: boolean
   fitContainer?: boolean
+  disabled?: boolean
 }) {
+  if (disabled) return <>{children}</>
```
Comment (T-Damer) [inline src/components/ToolTip.tsx:89]: If the Email Proof is closed, we don't need to display the tooltip

File: src/components/proofs/List.tsx [hunk 10]
```diff
@@ -49,8 +49,8 @@ export default function () {
           bottomPadding={savedMessage?.height || 0}
         >
           <div className={innerScrollableBlock}>
-            <EmailProofList />
             <ERC721ProofsList />
+            <EmailProofList />
```
Comment (T-Damer) [inline src/components/proofs/List.tsx:53]: They should be under `ERC721` proofs, sorry to include everything in one PR, I am falling asleep 😴

File: tailwind.config.js [hunk 11]
```diff
@@ -97,8 +109,8 @@ module.exports = {
         '-24': '-6rem',
         '-4': '-1rem',
       },
-      space: {
-        '-4': '-1rem',
+      rotate: {
```
Comment (smyzin) [inline tailwind.config.js:112]: Are sure that `space` with value `-4` doesn't need?

File: src/icons/Arrow.tsx [hunk 12]
```diff
@@ -1,93 +1,77 @@
...
+  width('w-full'),
+  height('h-auto'),
+  dropShadow('drop-shadow-secondary')
+)
 
 interface ArrowProps {
-  disabled?: boolean
-  turnDown?: boolean
-  vertical?: boolean
+  openDisabled?: boolean
```
Comment (smyzin) [inline src/icons/Arrow.tsx:33]: What do you think about `disabled` instead of `openDisabled`?

File: src/icons/Arrow.tsx [hunk 13]
```diff
@@ -1,93 +1,77 @@
...
+  pulseDisabled?: boolean
```
Comment (smyzin) [inline src/icons/Arrow.tsx:34]: Maybe it's better to call `pulse`?

File: src/components/ToolTip.tsx [hunk 14]
```diff
@@ -78,12 +78,16 @@ export default function ({
   arrow,
   fitContainer,
   children,
+  disabled,
```
Comment (smyzin) [inline src/components/ToolTip.tsx:81]: Strange prop for showing or not the Popup. But if it's okay, good then

**Review by smyzin — APPROVED** [review]
Need to polish a little but in general looks good

File: src/icons/Arrow.tsx (hunk 13 above)
Comment (T-Damer) [reply src/icons/Arrow.tsx:34]: Pulse is enabled by default, actually, let's disable it and enable with `pulse`

**Pull request merged**